lib/netext/grpcext/conn_test.go

```diff
@@ -24,26 +24,70 @@ import (
 )
 
 func TestInvoke(t *testing.T) {
-	t.Parallel()
+	ctx := context.Background()
```

Comment:
I believe the https://github.com/grafana/k6/blob/master/js/modules/k6/grpc/client_test.go is the better test for testing this, and we might need to introduce one more test service, which could contain a different method to test the wrappers.